backend-opal/migration.js

```diff
@@ -1,6 +1,11 @@
+/* eslint-disable global-require, no-unused-expressions, no-bitwise */
+(process.env.NODE_ENV === 'development' &&
+  require('dotenv').config({ path: `${__dirname}/.env.dev` })) |
```

Comment:
@hondahmd instead of this weird if-else:
```suggestion
  require('dotenv').config({ path: `${__dirname}/.env.${process.env.NODE_ENV}` })) |
```

Reply:
Yeah, just try to use the '|' operator. I've changed it.